lensterxyz/lenster
# feat: keep track of partially composed messages

## What does this PR do?

Fixes #1007
Screen recording: https://user-images.githubusercontent.com/667227/212549701-e724b33e-e40b-4ffb-a59b-27b5d708d7e0.mp4

Note: Using the same localstorage key for all messages. It would also be possble to store each message as a separate key. Then no JSON parsing would be needed, and no need to read and write all messages to update just one message. But I think that there will probably not be too many messages on the same time, so I went with just one localstorage key.

## Type of change


## How should this be tested?

Type a message in one conversation, without sending the message. Navigate to another conversation and then back to the first one. Observe that the message is restored.
